pyquil/paulis.py

```diff
@@ -119,7 +119,7 @@ def id(self, sort_ops=True):
         else:
             return ''.join("{}{}".format(p, q) for q, p in self._ops.items())
 
-    def operations_as_set(self):
+    def operations_as_set(self) -> FrozenSet[Tuple[int, str]]:
```

Comment:
I think the docstring just below is wrong. It says it returns a `frozenset of (op_str, coefficient)` pairs, but your return type annotation looks correct. I think the docstring should say `frozenset of (qubit, op_str)` pairs?

Reply:
Fixed.